roles/arteria-sequencing-report-ws/tasks/install_config.yml

```diff
@@ -0,0 +1,42 @@
+
+- name: ensure {{ arteria_service_name }} conf dir exists
+  file:
+    state: directory
+    path: "{{ arteria_service_config_root }}"
+
+- name: deploying {{ arteria_service_name }} app config
+  template:
+    src: seqreport_app.config.j2
+    dest: "{{ arteria_service_config_root }}/app.config"
+
+- name: deploying {{ arteria_service_name }} logger config
+  template:
+    src: seqreport_logger.config.j2
+    dest: "{{ arteria_service_config_root }}/logger.config"
+
+- name: deploying {{ arteria_service_name }} checkqc config
+  template:
```

Comment:
Since we don't substitute anything in the template we could add it to files instead and use `copy`.